airtime_mvc/public/js/airtime/library/library.js

```diff
@@ -236,16 +236,15 @@ var AIRTIME = (function(AIRTIME) {
                 "</div>"
             );
 
-        if (onDashboard) {
-            $menu.append(
-                "<div class='btn-group' title=" + $.i18n._('Publish') + ">" +
-                "<button class='btn btn-small' id='publish-btn'>" +
-                "<i class='icon-soundcloud-white'></i>" +
-                "<span>" + $.i18n._('Publish') + "</span>" +
-                "</button>" +
-                "</div>"
-            );
-        }
+            if (onDashboard) {
+                $menu.append(
+                    "<div class='btn-group' title=" + $.i18n._('Publish') + ">" +
```

Comment:
I think this button can go entirely since the publish feature doesn't work. I can open a separate pull request to remove the My Podcast feature.

Reply:
I intentionally only remove the soudcloud bits with this PR.

I'll do another one to get rid of podcast publishing if we decide to get rid of it.

With SoundCloud I was able to just search for "soundcloud" in the code and get rid of most of what I found. Getting rid of podcast publishing will be a bit more work since the best way to find everything is with a search for "publish" and that returns a lot of false postives.

Reply:
I thought we were looking into removing the My Podcast feature because it didn't work.

Yeah, it definitely is harder. Initial attempts to remove it keep breaking the interface. 😞